_posts/2020-01-29-cognitive-maps.md

```diff
@@ -0,0 +1,196 @@
+{: .text-left}
+
+Known as ["The Knowledge”](https://tfl.gov.uk/info-for/taxis-and-private-hire/licensing/learn-the-knowledge-of-london), all licensed cab drivers in London are required to memorize the city’s 25,000-plus streets as well as thousands of locations and landmarks. The goal is to instantaneously calculate the quickest route between two locations. This requirement might seem excessive in an age when handheld routing devices are in just about everyone's pocket. But, London’s street network is notorious for being one of the most difficult to navigate. The city's lack of grid-like layout can be a chore for the uninformed driver (Jacobs 2018).
+
+<div class="flex flex--around">
+  <figure class="caption">
+    <img class="caption__image" src="{% link images/grid_like_street_networks.png %}" alt="Street Network Orientation" />
+    <figcaption class="caption__text">The Complexity of Street Networks ( <a href="https://bigthink.com/strange-maps/charlotte-nc-has-americas-messiest-street-grid">Jacobs 2018</a> )</figcaption>
+  </figure>
+</div>
+
+So when Neuroscientist Eleanor Maguire set out to study the human brain’s internal navigation, there was no better place than an aspiring London cabbie (Maguire et al 2000).
+
+Because of their navigating abilities, she wondered if these cab drivers have a larger-than-average hippocampus. She got the idea after learning that the hippocampus is larger in animals (including the western scrub jays and squirrels) that stash their food and memorize the location to dig up later.
+
+Maguire’s study observed the drivers for four years as they prepared for the “The Knowledge” exam. Throughout this time, she and her colleagues took snapshots of their brain structure using Magnetic Resonance Imaging (MRI) scans. The results showed a sizable increase in the hippocampus in the drivers that passed the exam. Yet, in the control group, as well as in the candidates that failed the exam, there was no marked increase.
+
+<div class="flex flex--around">
+  <figure class="caption">
+    <img class="caption__image" src="{% link images/london_iconic_cabs.png %}" alt="London’s Black Cabs" />
+    <figcaption class="caption__text">London’s Iconic Black Cabs (<a href="https://www.ft.com/content/ccdde2f0-c0d0-11e4-876d-00144feab7de">Financial Times</a>)</figcaption>
+  </figure>
+</div>
+
+Additional brain navigation studies were conducted using a highly accurate and interactive virtual reality (VR) simulation of London’s central city as both cab drivers and non-cab drivers navigated the landscape using the Sony Playstation video game, The Gateway (Philos 2009). The game’s streets network was created using map data from the British Ordnance Survey and also simulated entities such as traffic signals, traffic, and pedestrians. Functional MRI (fMRI) scans taken while they navigated around VR London showed that brain activity in the hippocampus was active in both the licensed cab drivers and the non-cab drivers.
```

Comment:
I dont' see a philos 2009 source link at the bottom...

Reply:
good catch.  i'll add it.